content/en/platform/corda/5.0-beta/deploying/deployment-tutorials/manual.md

```diff
@@ -214,7 +214,14 @@ To create the schema manually, do the following:
        enabled: false
    ```
 
+{{< note >}}
+To apply SQL to a schema run the psql command line tool and use the format:
+`--dbname "dbname=cordacluster options=--search_path=CONFIG"`
```

Comment:
As a compromise, I'd suggest keeping the description of how to specify schema on `psql` in one place but I think it would be clearer if each of the "execute against the database" steps indicated which schema they should be executed against (which is DB agnostic).